### DISCUSSION TITLE
How to set google analytics tag?

### DISCUSSION THREAD

**Discussion by shungiku** [usememos/memos#3684] [Q&A]
Hi. I set this code in [Settings > System > Additional script] to enable google analytics tag in memos, but it didn't seem to work.
Does anyone know how can we set it or memos doesn't support it?

 ```javascript
</script>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=memomemos"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'memomemos');
</script>
<script>
```

In my browser it looks like this.
It seems that the code isn't recognized as a script block.
![image](https://github.com/user-attachments/assets/8db5cb80-ae86-48ca-a66d-8e4bd028b8a6)

memos version: ver0.22.2
environment: docker engine 27.0.3 on Ubuntu 24.04 LTS

**Comment by Priyanshu10230** [comment]
To set up the Google Analytics tag, you can [**implement Global Site Tag (gtag.js)**](https://www.tatvic.com/blog/gtag-js-future-tagging-set/?utm_campaign=forum_answer&utm_source=github&utm_medium=blog&utm_term=implement%20global%20site%20tag%20(gtag.js)). Follow these steps:

Sign in to Google Analytics: Log into your account. Sign in > Admin > Account & Property > Tracking Info > Tracking Code > Add Code Snippet - 

```
<!-- Google tag (gtag.js) -->
<script async src=""https://www.googletagmanager.com/gtag/js?id=UA-XXXXX-Y""></script>
<script>
 window.dataLayer = window.dataLayer || [];
 function gtag(){dataLayer.push(arguments);}
 gtag('js', new Date());

 gtag('config', 'UA-XXXXX-Y');
</script>
```

Replace UA-XXXXX-Y with your Google Analytics tracking ID. Place this code immediately after the opening <head> tag on every page of your site.

**Track Events -** 
- **For event tracking, use:**

```
<script>
 gtag('event', 'event_name', {
   'event_category': 'category',
   'event_label': 'label',
   'value': value
 });
</script>
```
